### DISCUSSION TITLE
global filter not working with custom cell

### DISCUSSION THREAD

**Discussion by shabeebcoder** [TanStack/table#2883] [Q&A]
{
    Header: 'Client Name ',
    Footer: 'Last Name',
   
    accessor: (props) => (
      <>
        <div class="text-sm text-gray-900">{props.clientname}</div>
        <div class="text-sm text-gray-600">{props.country}</div>
        <div class="text-sm text-gray-400">{props.clientstatus}</div>
      </>
    ),
  },

With custom cell (like above) global filter is not working 

if i change code above to following(below) then global filter work with this cell
```
```
  {
    Header: 'Client Name ',
    Footer: 'Last Name',
   accessor: 'clinetname"
  },

```

```
is there any way to make global filter work with custom cell ?

**Comment by akmur** [comment]
Hi, this is how I solved a similar issue:

https://codesandbox.io/s/react-table-basic-demo-with-header-groups-sort-filter-and-pagination-zoh4u?file=/src/components/columns.js

line 22

hope it helps

**Reply by shawngustaw** [reply]
@akmur the codesandbox link seems broken. Do you mind posting the code sample here?

**Comment by xkomiks** [comment]
Actual problem
